spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'quotes index page' do
+
+  before :each do 
+    allow(QuotesService).to receive(:search_quotes).and_return(JSON.parse(file_fixture("life_search_results.json").read, symbolize_names: true))
```

Comment:
Stubbing this was a really elegant choice. You don't need to rely on a third party gem for mocking a small amount of data, and you can control what the response is for accurate testing.